chapter-2.md

```diff
@@ -741,6 +741,37 @@ test "precision" {
 }
 ```
 
+# Advanced Function Invocation
+Zig offers a `@call` builtin function which gives you more flexibility with which how a function is
+invoked. There are options to enable or disable tail call optimizations, enable or disalbe inlining,
```

Comment:
Noticed there's a typo with the second "disable" on this line.

Reply:
Good catch, I'll update it.